app/src/Home.tsx

```diff
@@ -4,69 +4,21 @@ import AddJoke from "./AddJoke";
 import BackButton from "./BackButton";
 import NextButton from "./NextButton";
 
-interface Joke {
-  text: string;
-  complete: boolean;
-}
-
-interface HomeProps {
-  onClick: () => void;
-}
-
-export default function Home({ onClick }: HomeProps): JSX.Element {
+export default function Home(): JSX.Element {
   const [count, setCount] = useState<number>(0);
-  const newJokeToLaugh: Joke[] = [
```

Comment:
Good shout on removing the old code, thank you for cleaning up the Home component. ✨ 💖